frontend/src/components/employees/employees.module.scss

```diff
@@ -0,0 +1,50 @@
+.employees {
+  grid-area: main;
+  display: grid;
+  margin: 0 1em;
+  grid-template:
+    'header addbutton' 5em
+    'table table' auto
+    / auto 10em;
+}
+
+h1 {
+  grid-area: header;
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Can you list this font in the `constants` file, perhaps as `$table_font` or something?